docs/source/integrations/mlflow.md

```diff
@@ -0,0 +1,311 @@
+# How to add MLflow to your Kedro workflow
+
+[MLflow](https://mlflow.org/) is an open-source platform for managing the end-to-end machine learning lifecycle.
+It provides tools for tracking experiments, packaging code into reproducible runs, and sharing and deploying models.
+MLflow supports machine learning frameworks such as TensorFlow, PyTorch, and scikit-learn.
+
+Adding MLflow to a Kedro project enables you to track and manage your machine learning experiments and models.
+For example, you can log metrics, parameters, and artifacts from your Kedro pipeline runs to MLflow, then compare and reproduce the results. When collaborating with others on a Kedro project, MLflow's model registry and deployment tools help you to share and deploy machine learning models.
+
+## Prerequisites
+
+You will need the following:
+
+- A working Kedro project in a virtual environment. The examples in this document assume the `spaceflights-pandas-viz` starter.
+  If you're unfamiliar with the Spaceflights project, check out [our tutorial](/tutorial/spaceflights_tutorial).
+- The MLflow client installed into the same virtual environment. For the purposes of this tutorial,
+  you can use MLflow {external+mlflow:doc}`in its simplest configuration <tracking>`.
+
+To set yourself up, create a new Kedro project:
+
+```
+$ kedro new --starter=spaceflights-pandas-viz --name spaceflights-mlflow
+$ cd spaceflights-mlflow
+$ python -m venv && source .venv/bin/activate
+(.venv) $ pip install -r requirements.txt
+```
+
+And then launch the UI locally from the root of your directory as follows:
+
+```
+(.venv) $ pip install mlflow
+(.venv) $ mlflow ui --backend-store-uri ./mlflow_runs
```

Comment:
Why not leaving the default location? Well, because it's called `mlruns`, which could be conflated with https://github.com/mlrun/mlrun 😬 This requires us to write a `mlflow.yml` in the first step, but I think it's not the end of the world